spec/syntax.md

```diff
@@ -755,6 +756,8 @@ literal          = quoted-literal / unquoted-literal
 quoted-literal   = "|" *(quoted-char / escaped-char) "|"
 unquoted-literal = 1*name-char
 ```
+The **_<dfn>value of a literal</dfn>_** is the text after removing the terminating | characters if the literal is quoted,
```

Comment:
```suggestion
The **_<dfn>string value</dfn>_** of a literal is the text after removing the terminating `|` characters if the literal is quoted,
```

Reply:
Terminating doesn't seem right, since one of them is before the string starts. Literal is a term. Perhaps:

```suggestion
The **_<dfn>string value</dfn>_** of a _literal_ is the text content of that _literal_
not including any enclosing `|` characters of a _quoted literal_
and after unescaping any escaped characters.
```

Reply:
I like that phrasing better. However, my wording was a bit off; unquoted literals don't contain escaped characters. So probably clearer as the following:

The **_<dfn>string value</dfn>_** of a _literal_ is:
- for _unquoted literals_,  the text content of that _literal_
- for _quoted literals_,  text content of that _literal_, after removing the enclosing `|` characters and unescaping any escaped characters.

Reply:
It's true that unquoted literals cannot contain escaped characters, but that just means that "any escaped characters" is always an empty set, as it'll also be in most cases for quoted literals as well.

I'm also not opposed to forking the definition as proposed.

Reply:
I think that formally we are safe because \ can't be in unquoted literals, but I like splitting it out.

Reply:
```suggestion
The **_<dfn>string value</dfn>_** of a _literal_ 
for _unquoted literals_ is the text content of that _literal_;
or for _quoted literals_, the text content of that _literal_ 
after removing the enclosing `|` characters
and unescaping any escaped characters
```